AWSAutoScalingTests/AWSAutoScalingTests.m

```diff
@@ -26,7 +26,8 @@ @implementation AWSAutoScalingTests
 
 + (void)setUp {
     [super setUp];
-    [AWSTestUtility setupCognitoCredentialsProvider];
+    // Setup session based credentials to use for tests.
+    [AWSTestUtility setupSTSBasedSessionCredentialsProvider];
```

Comment:
nit, here and throughout:

1. change to `setupSessionCredentialsProvider`
1. Remove comment. It's perfectly clear without it

Reply:
done